tests/testthat/test_paramtest_randomPlantedForest_classif_rpf.R

```diff
@@ -0,0 +1,28 @@
+test_that("classif.rpf train", {
+  learner = lrn("classif.rpf")
+  fun = randomPlantedForest:::rpf.data.frame
+  exclude = c(
+    "x", "y", # handled internally
+    "max_interaction_ratio",
+    "max_interaction_limit"
+  )
+
+  # note that you can also pass a list of functions in case $.train calls more than one
+  # function, e.g. for control arguments
+  paramtest = run_paramtest(learner, fun, exclude, tag = "train")
+  expect_paramtest(paramtest)
+})
+
+test_that("classif.rpf predict", {
+  learner = lrn("classif.rpf")
+  fun = predict # nolint
```

Comment:
shouldn't we but the method here instead of the generic?

Reply:
`randomPlantedForest:::predict.rpf` is unexported, is that an issue?

Reply:
Nope, as long as you add `# nolint`

Reply:
Ah, right! Already changed that in current version.